distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -749,9 +753,9 @@ type File
                     matcher.matches (Path.of pathStr)
 
     ## GROUP Metadata
-       Checks if `self` is a child path of `other`.
-    is_child_of : File -> Boolean
-    is_child_of self other = self.starts_with other
+       Checks if `self` is a descendant of `other`.
+    is_descendant_of : File -> Boolean
+    is_descendant_of self other = self.starts_with other
```

Comment:
I renamed, because it does not check if the file is an _immediate child_ but it also returns True if the child is much deeper in the hierarchy, where the name 'child' does not seem to be suiting anymore - 'descendant' felt more fitting.